.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: ruby
 rvm:
-  - 2.1.0
+  - 2.4.0
+  - 2.3.3
+  - 2.2.6
+  - 2.1.10
```

Comment:
Your comment is good. But, now branching nokogiri version depending on Ruby version.
Please let me add this 🙏